### PR TITLE
Lots of little improvements & bugfixes

### PR DESCRIPTION
**Pull request by nicolaswurtz**
- removed darkmode on simulation (causing artefacts, to be refactored with SCSS refacto)
- removed Timetable in simulation & STDCM (became useless with new DriverTrainSchedule)
- corrected errors on DriverTrainSchedule
- some css fixes
- corrected bug on adding via by clicking on map
- added DriverTrainSchedule for STDCM result

### CODE REVIEW

File: front/src/applications/osrd/components/DriverTrainSchedule/DriverTrainScheduleModal.js [hunk 1]
```diff
@@ -41,7 +46,7 @@ export default function DriverTrainScheduleModal(props) {
           <div className="row no-gutters">
             <div className="col-4 col-xl-5">{t('drivertrainschedule:mass')}</div>
             <div className="font-weight-bold text-primary col-8 col-xl-7">
-              {rollingStockSelected.mass}T
+              {Math.round(rollingStockSelected.mass / 100) / 10}T
```
Comment (anisometropie) [inline front/src/applications/osrd/components/DriverTrainSchedule/DriverTrainScheduleModal.js]: What does that do ?
make a function that abstracts the behavior with an explicit name that says what it does.

**Review by anisometropie — CHANGES_REQUESTED** [review]

File: front/src/applications/osrd/components/DriverTrainSchedule/DriverTrainScheduleModal.js (hunk 1 above)
Comment (nicolaswurtz) [reply front/src/applications/osrd/components/DriverTrainSchedule/DriverTrainScheduleModal.js]: It represents mass with one decimal. Function created, done  ;)

File: front/src/applications/osrd/views/OSRDSimulation/DriverTrainSchedule.js [hunk 2]
```diff
@@ -35,12 +35,14 @@ export default function DriverTrainSchedule(props) {
     if (data) getRollingStock();
   }, [data]);
 
-  return (
+  return isModal ? (
     <ModalSNCF className="modal-drivertrainschedule" htmlID="driverTrainScheduleModal" size="xl">
       <ModalBodySNCF>
         {data && rollingStockSelected ? (
```
Comment (anisometropie) [inline front/src/applications/osrd/views/OSRDSimulation/DriverTrainSchedule.js]: if the second expression is empty, no need for a ternary, you can simplify
`data && rollingStockSelected && <Component />`

File: front/src/applications/osrd/views/OSRDSimulation/DriverTrainSchedule.js [hunk 3]
```diff
@@ -50,13 +52,23 @@ export default function DriverTrainSchedule(props) {
         </div>
       </ModalFooterSNCF>
     </ModalSNCF>
+  ) : (
+    <span>
+      {data && rollingStockSelected ? (
```
Comment (anisometropie) [inline front/src/applications/osrd/views/OSRDSimulation/DriverTrainSchedule.js]: same as above

**Review by anisometropie — CHANGES_REQUESTED** [review]

File: front/src/applications/osrd/views/OSRDSimulation/DriverTrainSchedule.js (hunk 3 above)
Comment (nicolaswurtz) [reply front/src/applications/osrd/views/OSRDSimulation/DriverTrainSchedule.js]: Simplified, done.

File: front/src/applications/osrd/views/OSRDSimulation/DriverTrainSchedule.js (hunk 2 above)
Comment (nicolaswurtz) [reply front/src/applications/osrd/views/OSRDSimulation/DriverTrainSchedule.js]: It's not really empty, <ModalBodySNCF /> needs a children, so '' is not nothing and it stops complaining :)

**Review by anisometropie — APPROVED** [review]

**Pull request merged**